src/domain/service.ts

```diff
@@ -5,7 +5,7 @@ import { AppConfig, getAppConfig } from '../config';
 import { ApplicationDocument, ApplicationModel } from './model';
 import 'moment-timezone';
 import moment, { unitOfTime } from 'moment';
-import _, { chunk } from 'lodash';
```

Comment:
i imagine its from TS import... no need to import `_` and the individual funcs. can remove `_`

Reply:
i think this was a just a difference in import style between bashar and i